interapp-backend/package.json

```diff
@@ -43,9 +43,8 @@
   "dependencies": {
     "cookie-parser": "^1.4.6",
     "cors": "^2.8.5",
-    "express": "^4.19.2",
-    "express-async-errors": "^3.1.1",
-    "express-rate-limit": "^7.1.4",
+    "express": ">=5.0.0",
```

Comment:
change to ^ from >= to ensure major version compat

Reply:
Done!